### PR TITLE
Issue158

### PR DESCRIPTION
**Pull request by amritanarasimhan**
Dragging the mouse up and down zooms in and out. However, I'm having a problem where if you mouse down on a control and drag, that control is not recognized as selected so dragging the control zooms in and out as well. I was wondering if there was anything to identify when control has been selected other than selectedcontrols?

### CODE REVIEW

File: packages/tools/guiEditor/src/diagram/workbench.tsx [hunk 1]
```diff
@@ -1025,6 +1027,19 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
...
+    }
+
+    zoomDrag(event: React.MouseEvent) {
+        let delta = 0;
+        if (event.movementY < 0) {
+            delta = -event.movementY;
+        }
+        if (event.movementY > 0) {
+            delta = -event.movementY;
+        }
```
Comment (RaananW) [inline packages/tools/guiEditor/src/diagram/workbench.tsx:1041]: I might be missing something, but both of those seem to do the same?

**Review by RaananW — COMMENTED** [review]
LGTM! just one question

File: packages/tools/guiEditor/src/diagram/workbench.tsx (hunk 1 above)
Comment (amritanarasimhan) [reply packages/tools/guiEditor/src/diagram/workbench.tsx:1041]: You're right! Not sure why I have those as two different conditionals.

**Review by RaananW — APPROVED** [review]
Discussed offline, one small change is needed, otherwise good to go!

**Pull request merged**